core/frontend/src/components/ethernet/InterfaceCard.vue

```diff
@@ -112,6 +151,37 @@ export default Vue.extend({
           notifications.pushError({ service: ethernet_service, type: 'ETHERNET_ADDRESS_DELETE_FAIL', message })
         })
     },
+    async triggerForDynamicIP(): Promise<void> {
+      ethernet.setUpdatingInterfaces(true)
+
+      await back_axios({
+        method: 'post',
+        url: `${ethernet.API_URL}/dynamic_ip`,
+        timeout: 10000,
```

Comment:
10 seconds is a pretty long time...

Reply:
Hue
This should take around 1-2 seconds usually (the interface is restarted, basically), but I'm basically using at least 10 seconds on all timeouts because of network issues, mostly which I recon happening on wifi connections.
I don't remember though if those tests were done before or after the introduction of the backend status checker.